ui/intermittent-failures/GraphAlternateView.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import ReactTable from 'react-table';
+import PropTypes from 'prop-types';
+
+const GraphAlternateView = ({ className, graphData, colNum, title }) => {
+  const columnsOne = [
```

Comment:
These variables are a little confusing. Technically this is two columns and `columnsTwo` is three columns total... can you rename them to `twoColumns` and `threeColumns` respectively?